### PR TITLE
setHistory dropping follow up, cleanup spawnHelpers

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Comment by martinpitt** [comment]
test/static-code says "no"

**Comment by jelly** [comment]
> test/static-code says "no"

Woops, fixed!

File: src/cockpit.d.ts [hunk 1]
```diff
@@ -176,6 +176,8 @@ declare module 'cockpit' {
         options: FileOpenOptions & { binary: true; syntax: FileSyntaxObject<T, Uint8Array>; }
     ): FileHandle<T>;
 
+    /* === cockpit.spawn ========================== */
+
```
Comment (martinpitt) [inline src/cockpit.d.ts:180]: Premature?

**Review by martinpitt — CHANGES_REQUESTED** [review]
Sending back as requested :grin:

**Review by martinpitt — APPROVED** [review]

**Pull request merged**